### DISCUSSION TITLE
Changing the timezone in a fluentd statefulset

### DISCUSSION THREAD

**Discussion by mikromas** [fluent/fluentd#4517] [General]
Hi!

I have a statefulset of fluentd.
I have changed timezone of the sts by editing the kubernetes sts.
After this change and the restart of the pods, curl towards fluentd gives the below: 

```
* Trying 10.128.14.78:9000...
* connect to 10.128.14.78 port 9000 failed: Connection refused
* Failed to connect to 10.128.14.78 port 9000 after 1 ms: Couldn't connect to server
* Closing connection
curl: (7) Failed to connect to 10.128.14.78 port 9000 after 1 ms: Couldn't connect to server
tb8-nk-jc-zts1-ztslenvoylbs-5c94cc4d4-2dsmh:/Envoy$
tb8-nk-jc-zts1-ztslenvoylbs-5c94cc4d4-2dsmh:/Envoy$
tb8-nk-jc-zts1-ztslenvoylbs-5c94cc4d4-2dsmh:/Envoy$ exit
exit
command terminated with exit code 7
```

Also there is no other log in fluentd logs indicating an issue and restart of the pods doesnt help.
Can timezone change make fluentd not to respond?

Thank you in advance!

**Comment by mikromas** [comment]
any update?

**Reply by daipom** [reply]
Looks like not able to connect in the first place.
Is Fluentd running and listening to `10.128.14.78 port 9000` correctly?
Do you have logs of Fluentd?
